### DISCUSSION TITLE
Teacher verification process is inaccessible and not working

### DISCUSSION THREAD

**Discussion by espertusnu** [community/community#42255] [GitHub Education]
I'm frustrated with the verification process for teacher benefits on GitHub.

First, the only way to upload supporting documentation is by taking a picture with my webcam or smart phone; there is no option to directly upload a PDF or image, so I printed my offer letter and then held it up to the camera. I have a fine motor disorder so can't hold a piece of paper steady.  I did my best but don't know if the picture will be too blurry for them to approve. I don't normally feel disabled, since I can type and use a mouse, but GitHub requires a working printer, ink, paper, and a steady hand. Surely I can't be the only person who does not have all of these.

Second, I am getting a message that my application cannot be reviewed until I complete my GitHub user profile with my full name and a short bio. I've already done that.

Not being a paying customer, I cannot directly contact support. I can only hope for help here. 

**Edit** I see that the latter is a known bug that others have posted about.

**Comment by StephenAnimates** [comment]
You are not alone, and I bought I'd contribute on your posting. I do not understand why there is no upload for verification. I am unwilling to use a webcam, or my phone camera, to capture an image of a document that is already provided digitally. Why am I to waste paper printing the verification, just to take a photo and upload. I find this is a step backward for GitHub. I'd rather not have educational access rather than allow a website access to my camera or have to use a webcam.

**Comment by WJC-POG** [comment]
I'm in the same boat as you guys. I've applied almost 20 times but keep getting rejections. I've tried uploading my work documents, course schedule, and employment contract. I am no longer sure what other official documents I need to provide to prove that I am a university teacher.
BTW, You can upload images by dragging and dropping them.
